pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[project]
+name = "nebula3-python"
+version = "3.4.0"
+description = "Python client for NebulaGraph V3.4"
+authors = [
+    {name = "vesoft-inc", email = "info@vesoft.com"},
+]
+dependencies = [
+    "future >= 0.18.0",
+    "httplib2 >= 0.20.0",
+    "pytz >= 2021.1",
+    "six >= 1.16.0",
+]
+requires-python = ">=3.7"
+readme = "README.md"
+license = {text = "Apache 2.0"}
+
+[project.urls]
+Homepage = "https://github.com/vesoft-inc/nebula-python"
+
+[build-system]
+requires = ["pdm-backend"]
+build-backend = "pdm.backend"
+
+[tool.pdm]
+package-type = "library"
+
+[tool.pdm.build]
+includes = ["nebula3"]
+
+[tool.pdm.dev-dependencies]
+dev = [
+    "black==22.8.0",
+]
+test = [
+    "pytest; python_full_version >= \"3.7.1\"",
+    "pytest-cov; python_full_version >= \"3.7.1\"",
```

Comment:
Maybe we have to give a version for pytest-cov?

ref: https://github.com/nedbat/coveragepy/issues/1376

Reply:
Upgraded the version